frontend/components/messages_components/ChatInput.jsx

```diff
@@ -18,10 +32,11 @@ const ChatInput = ({socket, user})=>{
           onPress={(event)=>{
             event.preventDefault();
             console.log(message);
```

Comment:
I believe it's fine to have console.logs in our code during development for visibility, but please default to what Kate advises in her review. Please remove them prior to production. Thank you!

Reply:
Awesome, thanks Kally. Yeah for this one I would still like to keep it since it's easy for me to check whether the message is coming from/sending to the correct user